src/System.Linq/src/System/Linq/Lookup.cs

```diff
@@ -182,7 +196,7 @@ internal List<TResult> ToList<TResult>(Func<TKey, IEnumerable<TElement>, TResult
             return _count;
         }
 
-        public IEnumerable<TResult> ApplyResultSelector<TResult>(Func<TKey, IEnumerable<TElement>, TResult> resultSelector)
+        public IEnumerator<TResult> ApplyResultSelector<TResult>(Func<TKey, IEnumerable<TElement>, TResult> resultSelector)
```

Comment:
Yes, that was foolish of me. I seem to have a mental block about thinking of this class as internal and treating changes to it as such. Probably because I normally only use it through the interface, so when I'm using rather than contributing I treat it as an implementation detail I don't then care about.